### PR TITLE
Remove multiple coordinators mention in Memory connector docs

### PR DESCRIPTION
**Pull request by konovenski**
## Description

Remove multiple coordinators mention in Memory connector docs

## Additional context and related issues

https://trinodb.slack.com/archives/C0305TQ05KL/p1730262972195419?thread_ts=1730262846.088699&cid=C0305TQ05KL

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
## Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Review by mosabua — APPROVED** [review]

**Comment by mosabua** [comment]
Please submit a signed CLA to enable a merge.

File: docs/src/main/sphinx/connector/memory.md [hunk 1]
```diff
@@ -101,6 +101,3 @@ in the catalog file.
   or may return partial data.
 - When the coordinator fails/restarts, all metadata about tables is
   lost. The tables remain on the workers, but become inaccessible.
-- This connector does not work properly with multiple
```
Comment (ebyhr) [inline docs/src/main/sphinx/connector/memory.md:104]: > Removed multiple coordinators mention in Memory connector docs

Could you avoid past tense in a commit title? https://cbea.ms/git-commit/ is our commit message guideline.

**Review by ebyhr — APPROVED** [review]

File: docs/src/main/sphinx/connector/memory.md (hunk 1 above)
Comment (ebyhr) [reply docs/src/main/sphinx/connector/memory.md:104]: @konovenski I mean a commit title, not PR title. We usually use `Rebase and merge` in this project.

Comment (konovenski) [reply docs/src/main/sphinx/connector/memory.md:104]: got it. done!

**Pull request closed without merging**